### PR TITLE
[wip] show endpoint

### PR DESCRIPTION
**Pull request by jitendra-1217**
### TODOs:
- Write tests for following very carefully, covering as much edge cases as possible:
  - [x] Move endpoint
  - [ ] Show endpoint
- Implement:
  - [ ] In show flow, check if all points made, if so mark game completed(broadcast event and etc.)
  - [ ] Verify route entries, Ensure game is running state for move/show endpoints(eg.)
  - [ ] Fix any TODO s added in code
  - [ ] Update toArray() method of models, add new attributes
  - [ ] Update postman collection
  - [ ] In nextTurn assignment, if user had no cards left, skip him. Such message to be broadcasted too.
  - [ ] Make a very simple single page(minimal) interface and play the game. Test thoroughly.
  - [ ] A job will query redis hash every sec(interval K1) by status and lastMoveMadeTimestamp (diff K2) and update nextTurn to prevTurn and publish this message to everyone.

### CODE REVIEW

**Comment by jitendra-1217** [comment]
screen shot 2016-12-05 at 12 18 19 am

File: src/AppBundle/Controller/GameController.php [hunk 1]
```diff
@@ -106,6 +103,38 @@ public function moveFromAction(
...
+    {
+        $this->init();
+
+        list($game, $user) = $this->gameService
+                                  ->getAndValidate(
+                                        $this->gameId,
+                                        $this->userId
+                                    );
+
+        list($success, $payload1, $payload2) = $this->gameService
```
Comment (Raghav-Sao) [inline src/AppBundle/Controller/GameController.php:119]: give pair of user and cards (which user have which card)

File: src/AppBundle/Model/Redis/Game.php [hunk 2]
```diff
@@ -180,4 +216,22 @@ public function toArray()
...
+    public function incrPoint(
+        string $userId,
+        float  $point
+    )
+    {
+        // Increments points by given amount for given user
+
+        $property = $this->getSNByUserId($userId) . "Points";
+
+        $this->$property += $point;
```
Comment (Raghav-Sao) [inline src/AppBundle/Model/Redis/Game.php:232]: can use 1 point for each card instead of current point

**Review by Raghav-Sao — APPROVED** [review]
